about/versioning.md

```diff
@@ -0,0 +1,49 @@
+---
+hidden: true
+icon: code-merge
+---
+
+# Versioning
+
+FO is consistently one of the first packs to upgrade to new Minecraft versions, thanks to its efficient versioning model.
+
+Of course, the speed at which FO is updated depends on multiple factors, including but not limited to:
+
+* Mojang's releases: new features, breaking changes...
+* Mods' updates: Fabric (which usually updates very quickly), mods...
+* New additions or removals of mods, which need testing
+* The time FO's author has to work on updates
+
+Generally, the development version of FO, which may not include all mods of a stable release, is released as soon as possible, often every week or two. Once a release is published, important updates to mods are usually quickly available.
+
+## Version Numbers
+
+An FO version is indicated by three numbers `X.Y.Z`, and a suffix for development versions. The three numbers now indicate:
+
+<table><thead><tr><th width="100" align="center">Number</th><th width="150">Meaning</th><th>Description</th></tr></thead><tbody><tr><td align="center"><code>X</code></td><td>Major versions</td><td><strong>Starting from 7.y.z</strong>: mapped to Minecraft's <a href="https://minecraft.wiki/w/Game_drop">game drops</a> or <a href="https://minecraft.wiki/w/Java_Edition_version_history#Full_release">full releases</a><br><strong>From 1.y.z ~ 6.y.z</strong>: mapped to Minecraft's full releases from 1.16</td></tr><tr><td align="center"><code>Y</code></td><td>Minor versions</td><td>Minecraft patches or changes to FO</td></tr><tr><td align="center"><code>Z</code></td><td>Patches</td><td>Non-breaking changes, no mod changes</td></tr></tbody></table>
+
+The suffix is a term that approximately corresponds to the stages of Minecraft's development:
+
+|  Minecraft  |   FO   |  Suffix  |
```

Comment:
This could be a bit misleading, since the first FO alpha doesn't come out until the release is available